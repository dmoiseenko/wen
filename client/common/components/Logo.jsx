import React from 'react';
import PropTypes from 'prop-types';
import classnames from 'classnames';


export default function Logo({ large }) {
  const className = classnames('logo', {
    'logo--large': large
  });

  return (
    <div className={className}>
      wen
    </div>
  );
}

Logo.propTypes = {
  large: PropTypes.bool.isRequired
};

Logo.defaultProps = {
  large: false
};
