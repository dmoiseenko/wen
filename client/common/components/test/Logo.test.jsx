import React from 'react';
import renderer from 'react-test-renderer';

import Logo from '../Logo';


it('should render', () => {
  const tree = renderer.create(
    <Logo />
  );

  expect(tree.toJSON()).toMatchSnapshot();
});

it('should render large logo', () => {
  const tree = renderer.create(
    <Logo large />
  );

  expect(tree.toJSON()).toMatchSnapshot();
});
