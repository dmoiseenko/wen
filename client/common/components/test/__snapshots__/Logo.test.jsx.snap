// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<div
  className="logo"
>
  wen
</div>
`;

exports[`should render large logo 1`] = `
<div
  className="logo logo--large"
>
  wen
</div>
`;
